examples/update-demo/2-create-replication-controller.sh

```diff
@@ -24,8 +24,8 @@ if [[ "${DOCKER_HUB_USER+set}" != "set" ]] ; then
 fi
 
 export KUBE_REPO_ROOT=${KUBE_REPO_ROOT-$(dirname $0)/../..}
-export KUBECFG=${KUBECFG-$KUBE_REPO_ROOT/cluster/kubecfg.sh}
+export KUBECTL=${KUBECTL-$KUBE_REPO_ROOT/cluster/kubectl.sh}
 
-set -x
```

Comment:
Another perhaps silly question: why remove `set -x`? This prints (to stdout?) all commands that are executed, correct? Do we not want this for logging?